terraform/gcp/security/gcp-org-member-default-service-account-iam-binding.yaml

```diff
@@ -0,0 +1,25 @@
+rules:
+    - id: gcp-org-member-default-service-account-iam-binding
+      patterns:
+        - pattern: resource
+        - pattern-inside: |
+              resource "google_organization_iam_binding" "..." {
+              ...
+              members = [..., $MEMBER, ...]
+              ...
+              }
+        - metavariable-regex:
+            metavariable: $MEMBER
+            regex: ((.*)-compute@developer.gserviceaccount.com)

```

Comment:
What part of this regex indicates that this is the "default" service account? I'm not familiar with GCP service accounts.

Reply:
The default service account structure is `PROJECT_NUMBER-compute@developer.gserviceaccount.com` (https://cloud.google.com/compute/docs/access/service-accounts#default_service_account).